flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
any chance to have it on aarch64? Like building from source?

Reply:
It is Electron, so probably not